Doctrine/Orm/Filter/SearchFilter.php

```diff
@@ -57,6 +57,16 @@ class SearchFilter extends AbstractFilter
     const STRATEGY_WORD_START = 'word_start';
 
     /**
+     * @var string Doctrine expression to case insensitivity comparison
+     */
+    const CASE_INSENSITIVE = 'LOWER';
```

Comment:
To be honest I think that this is useless to put it into a constant. That can't be changed by a developer and it's used only in one place.